src/Stratis.Bitcoin.Features.PoA/PoAFeature.cs

```diff
@@ -136,7 +136,9 @@ public override Task InitializeAsync()
             if (rebuildFederationHeight)
                 this.reconstructFederationService.Reconstruct();
 
-            this.miner?.InitializeMining();
+            // Initialize mining if the node is started in dev mode and is of role miner.
```

Comment:
This comment is a bit misleading, it implies that mining should only be started if that is the case